hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java

```diff
@@ -966,6 +966,9 @@ public class DFSConfigKeys extends CommonConfigurationKeys {
   public static final String  DFS_DATANODE_HTTP_ADDRESS_DEFAULT = "0.0.0.0:" + DFS_DATANODE_HTTP_DEFAULT_PORT;
   public static final String  DFS_DATANODE_HTTP_INTERNAL_PROXY_PORT =
       "dfs.datanode.http.internal-proxy.port";
+  public static final String DFS_DATANODE_NETTY_WORKER_NUM_THREADS_KEY =
+      "dfs.datanode.netty.worker.threads";
+  public static final int DFS_DATANODE_NETTY_WORKER_NUM_THREADS_DEFAULT = 0;
```

Comment:
Thanks for your contribution, but is it reasonable to default to 0 for the number of threads?

Reply:
@slfan1989 I have changed the default value to 10, and it works normally in our HDFS cluster.